developer-docs-site/docs/tutorials/full-node/run-a-fullnode.md

```diff
@@ -0,0 +1,310 @@
+---
+title: "Run a FullNode"
+slug: "run-a-fullnode"
+sidebar_position: 10
+---
+
+# Run a FullNode
+
+You can run your own [FullNode](/basics/basics-fullnodes) to synchronize with the state of the Aptos Blockchain and stay up-to-date. FullNodes replicate the entire state of the blockchain by querying other Aptos FullNodes or validators.
+
+Alternatively, you can use the FullNodes provided by Aptos Labs. However, such Aptos Labs-provided FullNodes have rate limits, which can impede your development. By running your own FullNode you can directly synchronize with the Aptos Blockchain and avoid such rate limits.
+
+FullNodes can be run by anyone. This tutorial explains how to configure a public FullNode to connect to the Aptos devnet.
+
+:::tip
+
+Your public FullNode will be connected to the Aptos devnet with a REST endpoint accessible on your computer at localhost:8080.
+
+:::
+
+## Before you proceed
+
+Before you get started with this tutorial, read the following sections:
+
+* [Validator node concepts](/basics/basics-validator-nodes).
+* [FullNode concepts](/basics/basics-fullnodes).
+* [REST specifications][rest_spec].
+
+:::caution Docker support only on Linux
+
+Docker container is currently supported only on Linux x86-64 platform. If you are on macOS or Windows platform, use the Aptos-core source approach.
+
+:::
+
+## Hardware requirements
+
+We recommend the following hardware resources:
+
+- For running a production grade FullNode:
+
+  - **CPU**: 4 cores (Intel Xeon Skylake or newer).
+  - **Memory**: 8GiB RAM.
+
+- For running the FullNode for development or testing:
+
+  - **CPU**: 2 cores.
+  - **Memory**: 4GiB RAM.
+
+## Storage requirements
+
+The amount of data stored by Aptos depends on the ledger history (length) of the blockchain and the number
+of on-chain states (e.g., accounts). These values depend on several factors, including: the age of the blockchain,
+the average transaction rate and the configuration of the ledger pruner.
+
+:::tip
+
+Given that devnet is currently being reset on a weekly basis, we estimate that Aptos will not require more than several GBs of storage. See the `#devnet-release` channel on Aptos Discord.
+
+:::
+
+## Configuring a FullNode
+
+You can configure a public FullNode in two ways:
+
+1. Using the Aptos-core source code.
```

Comment:
Maybe make this a link to https://github.com/aptos-labs/aptos-core, in which case do aptos-core, not Aptos-core.